.github/workflows/continuous-integration-workflow.yml

```diff
@@ -34,8 +34,8 @@ jobs:
       - name: Run unit tests
         run: npm run -w frontend test
 
-  frontend-e2e-test:
-    name: Frontend e2e testing
+  e2e-test:
```

Comment:
@frol JFYI
We're not testing e2e on production WAMP anymore, everything's local now.